pages/index.html

```diff
@@ -4,9 +4,25 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Index</title>
+  <link href="../styles/style.css" rel="stylesheet"/>
 </head>
+
 <body>
-  
+  <header>
+    <h1 class="page-title">Alie Ibarra</h1>
+    <h3 class="page-subtitle">A Digital Portfolio</h3>
```

Comment:
This isn't a heading (it wouldn't show up in a table of contents) and should probably be a `p` tag:

```suggestion
    <p class="page-subtitle">A Digital Portfolio</p>
```